adbe/adb_enhanced.py

```diff
@@ -948,22 +948,36 @@ def _get_all_packages(pm_cmd):
 
 
 # "dumpsys package" is more accurate than "pm list packages" but that means the results of
-# list_all_apps are now different from list_system_apps, list_non_system_apps, and
+# print_list_all_apps are now different from list_system_apps, list_non_system_apps, and
 # list_debug_apps
 # For now, we can live with this discrepancy but in the longer run we want to fix those
 # other functions as well
 # https://stackoverflow.com/questions/63416599/adb-shell-pm-list-packages-missing-some-packages
-def list_all_apps():
+def get_list_all_apps():
```

Comment:
I know I didn't use it earlier because of Python 2, but can we use the return syntax for these new functions?